sdk/storage/storage-blob/README.md

```diff
@@ -114,50 +115,46 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
-## SDK Architecture
-
-The Azure Storage SDK for JavaScript provides low-level and high-level APIs.
+## Examples
 
-- BlobServiceClient, ContainerClient and BlobClient objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
+### Create the Blob service Client
 
-- The high-level APIs provide convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).
-
-## Code Samples
+Use the constructor to create a instance of `BlobServiceClient`.
 
 ```javascript
-const {
-  BlobServiceClient,
-  newPipeline,
-  SharedKeyCredential,
-  AnonymousCredential,
-  TokenCredential
-} = require("@azure/storage-blob");
-
-async function main() {
   // Enter your storage account name and shared key
   const account = "account";
   const accountKey = "accountkey";
 
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
```

Comment:
Can you add a comment here to notice "SharedKeyCredential is only avaiable for Node.js runtime"? I got several StackOverflow questions asking why then cannot load this type in their React application.

Reply:
Will do.  BTW do you know why `SharedKeyCredential` only works in NodeJS? Because of `Buffer` usage?  Since connection string also relies on it, constructors taking connection strings won't work for browsers.

Reply:
@jeremymeng Root reason is storage doesn't want to customers using storage key in their browser session, it's not safe.